packages/web/src/pages/track-page/components/desktop/TrackPage.tsx

```diff
@@ -250,11 +250,12 @@ const TrackPage = ({
           css={{ maxWidth: 1080 }}
           justifyContent='center'
         >
-          {isCommentingEnabled ? (
+          {isCommentingEnabled && heroTrack?.owner_id ? (
```

Comment:
This is fine for now, but making a note for myself that this may interrupt the loading animation; may rework it later.